File: server/src/main/java/cn/iinti/atom/entity/SysConfig.java
package cn.iinti.atom.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.io.Serial;
import java.io.Serializable;
import java.time.LocalDateTime;

/**
 * <p>
 * 系统配置
 * </p>
 *
 * @author iinti
 * @since 2022-12-14
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
@TableName("sys_config")
@Schema(name = "SysConfig对象", description = "系统配置")
public class SysConfig implements Serializable {

    @Serial
    private static final long serialVersionUID = 1L;

    @Schema(name = "自增主建")
    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    @Schema(name = "key")
    private String configKey;

    @Schema(name = "value")
    private String configValue;

    @Schema(name = "创建时间")
    private LocalDateTime createTime;

    @Schema(name = "配置备注")
    private String configComment;


    public static final String ID = "id";

    public static final String CONFIG_KEY = "config_key";

    public static final String CONFIG_VALUE = "config_value";

    public static final String CREATE_TIME = "create_time";

    public static final String CONFIG_COMMENT = "config_comment";

}

File: server/src/main/kotlin/cn/iinti/atom/entity/SysConfig.kt
package cn.iinti.atom.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import io.swagger.v3.oas.annotations.media.Schema
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.experimental.Accessors
import java.io.Serial
import java.io.Serializable
import java.time.LocalDateTime

@Schema(name = "SysConfig对象", description = "系统配置")
@TableName("sys_config")
@Accessors(chain = true)
@Data
@EqualsAndHashCode(callSuper = false)
data class SysConfig(
    @Schema(name = "自增主建")
    @TableId(value = "id", type = IdType.AUTO)
    var id: Long? = null,

    @Schema(name = "key")
    var configKey: String? = null,

    @Schema(name = "value")
    var configValue: String? = null,

    @Schema(name = "创建时间")
    var createTime: LocalDateTime? = null,

    @Schema(name = "配置备注")
    var configComment: String? = null
) : Serializable {

    companion object {
        const val ID = "id"
        const val CONFIG_KEY = "config_key"
        const val CONFIG_VALUE = "config_value"
        const val CREATE_TIME = "create_time"
        const val CONFIG_COMMENT = "config_comment"
    }
}